scripts/upload_files.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -e
+
+usage() {
+  echo -n "##################################################
```

Comment:
I'm guessing that the ########### lines at the top and bottom are different lengths.